assignment3/day 5/githubSearchAPI.yaml

```diff
@@ -0,0 +1,111 @@
+swagger: '2.0'
+info:
+  description: >-
+    The Search API is optimized to help you find the specific item you're
+    looking for (e.g., a specific user, a specific file in a repository, etc.).
+    Think of it the way you think of performing a search on Google. It's
+    designed to help you find the one result you're looking for (or maybe the
+    few results you're looking for). Just like searching on Google, you
+    sometimes want to see a few pages of search results so that you can find the
+    item that best meets your needs. To satisfy that need, the GitHub Search API
+    provides up to 1,000 results for each search.
+  version: 3.0.0
+  title: GitHub Search
+  contact:
+    email: vmishra@grepruby.com
+host: api.github.com
+basePath: /
+tags:
+  - name: Search
+    description: >-
+      Find repositories via various criteria. This method returns up to 100
+      results per page.
+    externalDocs:
+      description: Find out more
+      url: 'https://developer.github.com/v3/search/'
+schemes:
+  - https
+paths:
+  /search/repositories:
+    get:
+      tags:
+        - Search
+      summary: Find repositories via various criteria.
+      consumes:
+        - application/json
+      produces:
+        - application/json
+      parameters:
```

Comment:
where are the query parameter for pagination? What if i want to fetch the next 200 items for the same search result?